apps/astarte_data_updater_plant/test/astarte_data_updater_plant/data_updater/core/trigger_handler_test.exs

```diff
@@ -0,0 +1,318 @@
+defmodule Astarte.DataUpdaterPlant.DataUpdater.Core.TriggerHandlerTest do
+  @moduledoc false
+  alias Astarte.DataUpdaterPlant.DataUpdater
+  alias Astarte.DataUpdaterPlant.DataUpdater.Core.Trigger
+  alias Astarte.Core.Triggers.SimpleTriggersProtobuf.DeviceTrigger
+  alias Astarte.Core.Triggers.SimpleTriggersProtobuf.DataTrigger
+  alias Astarte.Core.Triggers.SimpleTriggersProtobuf.SimpleTriggerContainer
+  alias Astarte.Core.Triggers.SimpleTriggersProtobuf.AMQPTriggerTarget
+  alias Astarte.DataUpdaterPlant.AMQPTestHelper
+  alias Astarte.Core.Triggers.SimpleTriggersProtobuf.TriggerTargetContainer
+
+  use Astarte.Cases.Data, async: true
+  use Astarte.Cases.Device
+  use ExUnitProperties
+
+  use Mimic
+
+  setup_all :populate_interfaces
+
+  setup_all %{realm_name: realm_name, device: device} do
+    {:ok, message_tracker} = DataUpdater.fetch_message_tracker(realm_name, device.encoded_id)
+
+    {:ok, dup} =
+      DataUpdater.fetch_data_updater_process(realm_name, device.encoded_id, message_tracker, true)
+
+    Astarte.DataAccess.Config
+    |> allow(self(), dup)
+
+    GenServer.call(dup, :start)
+
+    %{data_updater: dup}
+  end
```

Comment:
You can automate this as other tests are doing, see [here](https://github.com/astarte-platform/astarte/blob/16cbbbe5a2e0d1c93a079120f79d1340072abd6c/apps/astarte_data_updater_plant/test/astarte_data_updater_plant/data_updater/core/device_test.exs#L43-L48) for reference